2007-03-13  Simon Josefsson  <simon@josefsson.org>

        * libtelnet/shishi.c (krb5shishi_send): Don't set a
        use-session-key ap-options, that is for user2user authentication
        which is not appropriate here.

Index: b/libtelnet/shishi.c
===================================================================
--- a/libtelnet/shishi.c
+++ b/libtelnet/shishi.c
@@ -1,4 +1,4 @@
-/* Copyright (C) 2002, 2003 Simon Josefsson
+/* Copyright (C) 2002, 2003, 2007 Simon Josefsson
    Copyright (C) 2003 Free Software Foundation, Inc.
 
 This file is part of Shishi / GNU Inetutils.
@@ -169,10 +169,6 @@ krb5shishi_send (TN_Authenticator * ap)
   else
     ap_opts = 0;
 
-#ifdef ENCRYPTION
-  ap_opts |= SHISHI_APOPTIONS_USE_SESSION_KEY;
-#endif	/* ENCRYPTION */
-
   type_check[0] = ap->type;
   type_check[1] = ap->way;
 
